packages/@okta/vuepress-site/docs/guides/pwd-optional-change-email/main/react/integrationsteps.md

```diff
@@ -0,0 +1,139 @@
+### 1. The user signs in to your app
+
+The user signs in to your app before they can change their primary email address. During the sign-in flow create an `OktaAuth` object, which is required to authenticate the user and later change their email.
+
+```javascript
+const oktaAuth = (() => {
+  return new OktaAuth(oidcConfig);
+})();
+```
+
+ To learn more about initializing this object and how to integrate the user sign in, see [Sign in with email only](/docs/guides/pwd-optional-sign-in-email/nodeexpress/main/).
+
+<!-- Temporarily set to nodejs. The above react sign in guide will be completed in https://oktainc.atlassian.net/browse/OKTA-502075 -->
+
+### 2. The user starts the change primary email flow
+
+The user starts the change primary email flow by clicking an **Edit** link next to the primary email address field. Add an **Edit** link that gives the user an entry point to change their email.
+
+<div class="half border">
+
+![Screenshot showing an edit link next to the primary email address.](/img/pwd-optional/pwd-optional-change-email-my-account-js-react-edit-email.png)
+
+</div>
+
+### 3. The user submits a new primary email
+
+When the user clicks on the **Edit** link, display a dialog that allows the user to change and submit their new email address.
+
+<div class="half border">
+
+![Screenshot showing a page with a new primary email input field and continue button.](/img/pwd-optional/pwd-optional-change-email-my-account-js-react-submit-email.png)
+
+</div>
+
+1. When the user clicks **Continue** and submits their new email address, create an object of type [`AddEmailPayload`](https://github.com/okta/okta-auth-js/blob/master/docs/myaccount/modules.md#addemailpayload) and set
+
+* `profile.email` to the new primary email.
+* `role` to `PRIMARY` which identifies the email as the primary email address.
+* `sendEmail` to `true` which sends the email challenge to the newly added email. The default is `true`.
```

Comment:
can also mention another approach (call `sendEmailChallenge`) to send email when this option is set to false

Reply:
Added a note mentioning to call `sendEmailChallenge()` when `sendEmail` is set to false.